Updates to Reproduction number section (#351)
greenelab/covid19-review: content/07.pathogenesis.md

The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.

> Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1098/rspa.1927.0118; @doi:10.1038/280361a0].

Reviewer: ```suggestion
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
```

Good catch on the wrong year @alavendelm.  If we break convention and cite with the URL here, Zotero is able to extract the correct metadata:
```
$ manubot cite --render doi:10.1098/rspa.1927.0118
1. A contribution to the mathematical theory of epidemics
Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character
(1997-01) https://doi.org/fwx2qw
DOI: 10.1098/rspa.1927.0118

$ manubot cite --render https://doi.org/10.1098/rspa.1927.0118
1. A contribution to the mathematical theory of epidemics
William Ogilvy Kermack, A. G. McKendrick, Gilbert Thomas Walker
Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character (1927-08-01) https://royalsocietypublishing.org/doi/10.1098/rspa.1927.0118
DOI: 10.1098/rspa.1927.0118
```
Should Gilbert Thomas Walker be listed as an author or not?  The HTML source lists him as an author, which is what Zotero picks up.  The PDF says "communicated by" Walker and has only two authors.

Author: No Walker is not an author - "communicated by" him (I guess like PNAS direct submission, back in the day). Btw paper was first published online in 01/1997 so I guess that's what confused Zotero.

Reviewer: Got it. I'll make a commit in your fork to update the manual references so that Walker isn't included.

More generally, population-level epidemic dynamics can be both observed and modelled.
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs or the comparison of predicted outcomes among prevention and control strategies.
Many current efforts to model R~t~ have led to tools that assist the visualization of estimates in real time (or over recent intervals) [@url:https://epiforecasts.io/covid/; @url:https://rt.live/].

> While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region.

Reviewer 2: ```suggestion
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
```

> While these may be valuable resources, it is important to note that they should be met with caution: the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region.

Reviewer 3: Looks good. Maybe just soften the note about caution somehow? Some of the preprints that we cite in the paragraphs above seem to be using very similar methods as the two "real-time" websites but they are mentioned without comment.

We could remove the caution part and just state the potential limitations:
```suggestion
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region.
```   

We could keep the caution part but mention why we are adding it here but not to papers that used similar methods:
```suggestion
While these may be valuable resources, it is important to note that they should be met with caution: the estimates arise from models containing many assumptions; and rely on the data quality, which varies widely by region; although these factors can be an issue for any model, real time visualization may not have the same rigorous evaluation of these factors that retrospective academic research may have.
```

Reviewer 3: Also, should we put these two sentences at the end of the previous paragraph?

Author: thanks for your suggestions here - I take your point: why pick on 'live' estimates and not papers/preprints? I have no reason to think that the live models are of lesser quality than preprint estimates, but I think the public is far more likely to look at these live models -- with their nice visualization -- and I worry about public policy being informed by poorly understood modeling results. Thus my intent to express caution, though I do not want to veer into politics! Any help to achieve this end is appreciated.

Author: I think both your suggestions here are good; will wait for a little bit in case anyone else wants to chime in, before accepting them.
